packages/core/config/jest.config.js

```diff
@@ -0,0 +1,18 @@
+// used by cli `rw test` command
+
+const path = require('path')
+
+const { getPaths } = require('@redwoodjs/internal')
+
+const redwoodPaths = getPaths()
+
+module.exports = {
+  collectCoverageFrom: [
+    '**/*.{js,jsx,ts,tsx}',
+    '!**/node_modules/**',
+    '!**/dist/**',
+  ],
+  coverageDirectory: path.join(redwoodPaths.base, 'coverage'),
+  rootDir: redwoodPaths.base,
+  projects: ['<rootDir>/{,!(node_modules)/**/}jest.config.js'],
```

Comment:
Can you explain what this does?

Reply:
This searches your app for jest configs and executes all of those (or only the ones you passed into `--projects`).

Reply:
Is it possible that we could put these in `config/jest.config.js`?

Reply:
I'm not sure what you mean here...

Reply:
Re-reading this I'm not sure what I mean either! Time heals all wounds. #ignore this.